docs/user-guide-policy-configure-ovs-bridge.md

```diff
@@ -0,0 +1,86 @@
+# Tutorial: Create a Ovs Bridge and Connect it to a Node Interface
```

Comment:
Please add reference to https://github.com/nmstate/kubernetes-nmstate#deployment-and-usage and https://github.com/nmstate/kubernetes-nmstate/blob/master/docs/user-guide.md